static/css/perdiem.css

```diff
@@ -299,3 +299,22 @@ ul.tabs li {
     margin: 0 auto;
     padding: 20px;
 }
+
```

Comment:
Cool. How do I get it to load the new CSS file? Also, where should it be saved in the CSS folder?